tests/model_serving/model_server/authentication/utils.py

```diff
@@ -0,0 +1,111 @@
+import base64
+import re
+from contextlib import contextmanager
+from typing import Dict, List, Optional
+
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.role import Role
+from simple_logger.logger import get_logger
+
+from utilities.inference_utils import Inference
+
+LOGGER = get_logger(name=__name__)
+
+
+def base64_encode_str(text: str) -> str:
```

Comment:
This is already defined in https://github.com/opendatahub-io/opendatahub-tests/pull/22/files/38b1f728c8d82955805212d36cf8c47f03d17a10#diff-5045407005153d8609d7df4be7a249a67b6eebd4c8b970597fcecedce95fd2c5R101